ncovstats/src/country.js

```diff
@@ -57,6 +62,65 @@ const toTitleCase = (phrase) => {
     .join(' ');
 };
 
+
+const abbrExpand = (shortName) => {
+  longName = shortName;
+  switch (shortName) {
+    case 'Us':
```

Comment:
Is it possible to make the cases all caps? Like `US` because we can get rid of the function `toTittleCase`.

Reply:
We are getting data from a web page and it doesn't has `us` or `US` but `United States`, but User is free to type anything. So we need to convert the userInput and `toTittleCase` is required here.

Reply:
We need the country code expanded here to retrieve data, I'm I right?
```const countryStat = html(`td:contains(${country})`);```